src/main/java/org/testng/TestNG.java

```diff
@@ -43,6 +41,8 @@
 import org.testng.internal.thread.graph.SuiteWorkerFactory;
 import org.testng.junit.JUnitTestFinder;
 import org.testng.log4testng.Logger;
+import org.testng.remote.SuiteDispatcher;
```

Comment:
the remote package was moved to https://github.com/testng-team/testng-remote in PR #953 
is there any special reason you added it back?